api/src/test/java/org/openmrs/module/fhir2/providers/r3/ConditionFhirR3ResourceProviderTest.java

```diff
@@ -132,15 +145,35 @@ public void getConditionHistoryByWithWrongId_shouldThrowResourceNotFoundExceptio
 		assertThat(resourceProvider.getConditionHistoryById(idType).size(), Matchers.equalTo(0));
 	}
 	
-	//	@Test
-	//	public void shouldCreateNewCondition() {
-	//		when(conditionService.saveCondition(condition)).thenReturn(condition);
-	//
-	//		MethodOutcome result = resourceProvider.createCondition(condition);
-	//		assertThat(result, notNullValue());
-	//		assertThat(result.getCreated(), is(true));
-	//		assertThat(result.getResource(), equalTo(condition));
-	//	}
+	@Test
+	public void createCondition_shouldCreateNewCondition() {
+		when(conditionService.create(any(org.hl7.fhir.r4.model.Condition.class))).thenReturn(condition);
+		
+		MethodOutcome result = resourceProvider.createCondition(Condition30_40.convertCondition(condition));
+		
+		assertThat(result, notNullValue());
+	}
+	
```

Comment:
Please add a tests for missing id and not existent condition-
1. updateCondition_shouldThrowInvalidRequestExceptionForMissingId
2. updateCondition_shouldThrowMethodNotAllowedIfDoesNotExist